temporal/t.rast.algebra/testsuite/test_raster_algebra.py

```diff
@@ -258,6 +259,51 @@ def test_simple_arith_td_4(self):
         self.assertEqual(start, datetime.datetime(2001, 1, 1))
         self.assertEqual(end, datetime.datetime(2001, 1, 5))
 
+    def test_simple_arith_time_function(self):
+        """Simple arithmetic test of time functions"""
+
+        E = SimpleModule(
+            "t.rast.algebra",
+            expression="R = if(A >= 3, start_doy(A, 0), 0)",
+            basename="r",
+            flags="d",
+            suffix="gran",
+        )
+        E.run()
+        self.assertModule(
+            "t.rast.algebra",
+            expression="R = if(A >= 3, start_doy(A, 0), 0)",
+            basename="r",
+            suffix="gran",
+        )
+
+        D = tgis.open_old_stds("R", type="strds")
+
+        # Check expressions
+        ref_str = "...".join(
+            [
+                "r_2001_01_01=if(a1@... >= 3, 1, 0)",
+                "r_2001_01_02=if(a2@... >= 3, 2, 0)",
+                "r_2001_01_03=if(a3@... >= 3, 3, 0)",
+                "r_2001_01_04=if(a4@... >= 3, 4, 0)",
+            ]
+        )
+        ref_str = f"...{ref_str}..."
+        self.assertLooksLike(str(E.outputs.stdout), ref_str)
```

Comment:
My guess would be that this is an implementation detail, but I will let you decide that. In any case, I would do the test with `-d` in a separate test function. It looks like an additional, informative thing separate from producing the actual output.

Reply:
Separating the two tests revealed that both of them are failing. I guess that's good news and bad news. Anyway, this part is resolved.